examples/discovery-search-app/package.json

```diff
@@ -23,8 +22,8 @@
   },
   "dependencies": {
     "@carbon/icons": "^10.5.0",
-    "@ibm-watson/discovery-react-components": "file:../../packages/discovery-react-components",
-    "@ibm-watson/discovery-styles": "file:../../packages/discovery-styles",
+    "@ibm-watson/discovery-react-components": "^1",
+    "@ibm-watson/discovery-styles": "^1",
```

Comment:
From the [yarn docs](https://yarnpkg.com/features/workspaces):

> If the package manager was to resolve a range that a workspace could satisfy, it will prefer the workspace resolution over the remote resolution if possible.

I tried using `workspace:` protocol first, but that broke `lerna`. This approach here should mean that the example app _should_ pull in these packages from the local workspaces, rather than pulling from remote repo.